packages/drivers/odsp-driver/src/odspDelayLoadedDeltaStream.ts

```diff
@@ -161,6 +174,11 @@ export class OdspDelayLoadedDeltaStream {
 					!requestWebsocketTokenFromJoinSession,
 				);
 			}
+			if (websocketEndpoint.sensitivityLabelsInfo !== undefined) {
```

Comment:
It's not very clear what it means when it's undefined:
1. Today it likely means - service did not implement contract.
2. But when everything settles - what does it mean it's empty? Does it mean no labels?

Depending on second answer, code below might be incorrect. I.e. if it means all labels are gone, we need to raise event and tell to consumers there are no more labels out there - we can't just ignore this new information.

I'd put big comment above `sensitivityLabelsInfo` property definition on how to interpret undefined (and reduce chanced future developers will make mistakes). Semantics matter.

Reply:
@marcmasmsft @GaryWilber Can you answer this question, that could JoinSession Response still has undefined as labels once your change is done? Does files always have some labels or they could be empty? Also adding @jbissat